### PR TITLE
remove json/out files saved in /tmp/qtop_results_USER after X mins

### PR DESCRIPTION
**Pull request by sfranky**

### CODE REVIEW

File: fileutils.py [hunk 1]
```diff
@@ -116,15 +116,15 @@ def __init__(self, fn):
...
     """
     deletes older json files in savepath directory.
     experimental and loosely untested
     """
-    time_alive = int(config['auto_delete_old_json_files_after_x_minutes'])
+    time_alive = int(config['auto_delete_old_output_files_after_x_minutes'])
     user_selected_save_path = os.path.realpath(os.path.expandvars(config['savepath']))
     for f in os.listdir(user_selected_save_path):
-        if not f.endswith('json'):
+        if not f.endswith(('json', '.out')):
```
Comment (fgeorgatos) [inline fileutils.py]: very generic expression. could this start deleting other unrelated files? or is it safe due to dir?

Comment (sfranky) [inline fileutils.py]: exactly, it's not just in /tmp

**Comment by fgeorgatos** [comment]
lgtm, merged. 

Needs to be verified further, because of possible /tmp files leakage from other parts of the code.
